mesonbuild/cmake/interpreter.py

```diff
@@ -958,71 +979,72 @@ def analyse(self) -> None:
 
         # Find all targets
         added_target_names = []  # type: T.List[str]
-        for i in self.codemodel_configs:
-            for j in i.projects:
+        for i_0 in self.codemodel_configs:
+            for j_0 in i_0.projects:
                 if not self.project_name:
-                    self.project_name = j.name
-                for k in j.targets:
+                    self.project_name = j_0.name
+                for k_0 in j_0.targets:
                     # Avoid duplicate targets from different configurations and known
                     # dummy CMake internal target types
-                    if k.type not in skip_targets and k.name not in added_target_names:
-                        added_target_names += [k.name]
-                        self.targets += [ConverterTarget(k, self.env)]
+                    if k_0.type not in skip_targets and k_0.name not in added_target_names:
+                        added_target_names += [k_0.name]
+                        self.targets += [ConverterTarget(k_0, self.env)]
```

Comment:
gahh, the lack of scoping in python makes this so painful...

Reply:
I agree. I already tried coming up with better names in most instances but decided to give up here and just use numbers instead...